packages/ethereum-contracts/test/contracts/superfluid/Superfluid.test.ts

```diff
@@ -430,7 +438,9 @@ describe("Superfluid Host Contract", function () {
                     paNFTLogicAddress,
                     pmNFTLogicAddress
                 );
-                await governance.updateContracts(
+                await governance[
+                    "updateContracts(address,address,address[],address,address)"
```

Comment:
If you removed the overloadded updateContracts, are these changes still needed.

Reply:
no, shall we revert those?

Reply:
If you are sure about removing the old method.

I asked in another thread though,  are you sure? Have you  thought about all the testnets we are still supporting, and any mainnet that might still use the old abi.

Reply:
yes, I'm sure